### ISSUE TITLE
Option to set tool tip text size 

### ISSUE THREAD

**Issue by euu2021** [freeplane/freeplane#1728]
What:
I would like to have the option to set Tool tip text size. 

Why:
Currently, FP automatically set the tool tip text size as the size of the node core. There are already options to set the size of the tool tip, but not the actual text inside the tool tip. This creates a problem when the size of the node core is small (or big), because the tool tip text gets so small (or big) that it's basically useless.

In other words, the user can have nodes with text size very small, like 2, or very big, like 72, and work with it just fine, because it's possible to use zoom in/out. But, those small/big sizes get transferred to the tooltip, generating a tooltip that has text too small or too big:

Small example:
![image](https://github.com/freeplane/freeplane/assets/77707706/4c75a2a0-8c59-4d10-84b4-753e0beaefbe)

Big example:
![image](https://github.com/freeplane/freeplane/assets/77707706/80edde14-62ce-44fc-8190-821b88ad0217)

It would also allow more consistency if the user uses nodes with different text sizes.

TL;DR I would like to set the tooltip text to the size that I find ideal (size 14), no matter the size of the text of the node core.

Example map: 
[EXAMPLE tooltip size.zip](https://github.com/freeplane/freeplane/files/14539998/EXAMPLE.tooltip.size.zip)

**Comment by dpolivaev** [comment]
Why do you use node font sizes that are hard to read?

**Comment by euu2021** [comment]
> Why do you use node font sizes that are hard to read?

Some years ago I did some tests and concluded that a smaller size performed better. I don't remember why I came to this conclusion, it was something related to the rendering of elements like edges and borders.

Anyway, today I resized my nodes to a bigger size, and it seems to be working fine. So, my interest in this feature is on hold, while I test the bigger font size.

**Comment by euu2021** [comment]
I'm happy with the bigger fonts, so I'm closing this issue. Thanks,

**Comment by euu2021** [comment]
I need to reopen this, because using bigger fonts force me to use smaller zoom levels, which causes me a flickering problem with the nodes, as explained at https://github.com/freeplane/freeplane/issues/1744.

Now, maybe the font of the tooltip could be set according to a static variable, that the user can change at runtime (I don't know if this idea makes sense). Apparently, the tooltip font for that situation is defined at those lines:

https://github.com/freeplane/freeplane/blob/cb4ed89f740466664d070ad62e11c4e9195856e5/freeplane/src/main/java/org/freeplane/features/text/TextController.java#L432-L440

**Comment by dpolivaev** [comment]
@euu2021, could you please explain your use case? Freeplane uses fonts specified for appropriate elements, so I don’t understand why you’d need font sizes that are not comfortable to view on your screen.

Typically, if a 10pt or 12pt font doesn’t appear correctly at 100% zoom, it suggests an issue with the monitor’s pixel density calculation or settings, which should be addressed.

Please provide relevant mind maps and screenshots for further analysis.

**Comment by euu2021** [comment]
> could you please explain your use case?

I use "minimize node" a lot, so I usually read the node core using the tooltip. Sometimes, I just want to skim through the content in a zoomed out map, so a small font is ideal, because it shows more content in the tooltip, and is easier to skim through. But, sometimes, I want to read more carefully the content of the nodes, so I zoom in into the map, and them it is ideal to have a bigger font, because it is more comfortable to read.

So, it's a matter of transitioning between two different vibes: 
- fast/zoom out/small font; 
- slow/zoom in/big font

The zoom in/zoom out is easy to do; but the small font/big font is not. Changing the font for all the nodes in the map is annoying, because many other formatting settings are set in relation to the font size. So, it would be easier to simply change the font of the tooltip.

The ideal would be if the size of the font of the tooltip, could be changed by using Ctrl+MouseWheelUP/DOWN, and that size would be set also in future tooltips.

**Comment by dpolivaev** [comment]
Currently, Freeplane does not have a separate setting for tooltip font size; tooltips inherit the font size of their corresponding content. Since tooltips can display HTML-formatted content, adjusting their appearance requires careful consideration.

To better understand and address your issue, could you please provide more details about your formatting settings related to font size? Screenshots illustrating the problem would be particularly helpful.

If you’re looking to apply consistent font sizes across multiple nodes without adjusting each one individually, you can utilize Freeplane’s styling features. By creating and applying styles, you can manage font sizes and other formatting attributes efficiently.
